content/docs/context.md

```diff
@@ -239,6 +240,10 @@ It is often necessary to update the context from a component that is nested some
 **app.js**
 `embed:context/updating-nested-context-app.js`
 
+### Usage with useContext
+
+https://reactjs.org/docs/hooks-reference.html#usecontext
```

Comment:
We should probably move this to an embed like the other context examples, but it would be good to show an inline example here. The one on the Hooks page is consistent with the example context already on this page (a "theme" provider).

```suggestion
### Usage with the useContext Hook

```jsx
function ThemedButton() {
  const theme = useContext(ThemeContext);
  return (
      <button style={{ background: theme.background, color: theme.foreground }}>
      I am styled by theme context!
    </button>
  );
}
```

[useContext Hook API Reference](/docs/hooks-reference.html#usecontext)
```

Reply:
You're probably right. I didn't write anything as I felt it would only be redundant, and I'm not much of a technical writer anyway. 

For a moment I thought that I couldn't use context in function components, because this page didn't even mention them, and most answers I found via Google were either wrong or written in the React.Component era.

I feel like just adding a link would be enough to mitigate that problem, but maybe more is more in this case.